src/build-data/policy/bsi.txt

```diff
@@ -153,8 +164,10 @@ sm2
 # pk_pad
 #eme_pkcs1 // needed for tls
 #emsa_pkcs1 // needed for tls
+eme_raw
```

Comment:
Don't prohibit, just not part of the default module set, please.

Reply:
From what I understand it is not strictly needed by any other module. Though, it allows applications to implement their own RSA encryption padding scheme. Basically, do the padding yourself and then encrypt with "EME Raw". No reason to prohibit that.

Reply:
I agree the `needed` is misleading here. I'm not aware of any module there `eme_raw` and `raw_hash` are needed.
It's just that we don't want to prohibit them in case a user needs them to implement a special (corner) case.